### PR TITLE
docs: Add documentation on how to sign image-based gadgets.

### PR DESCRIPTION
**Pull request by eiffel-fl**

### CODE REVIEW

File: docs/devel/hello-world-gadget.md [hunk 1]
```diff
@@ -207,6 +207,12 @@ Pushing ghcr.io/my-org/mygadget:latest...
 Successfully pushed ghcr.io/my-org/mygadget:latest@sha256:dd3f5c357983bb863ef86942e36f4c851933eec4b32ba65ee375acb1c514f628
 ```
 
+## Signing the gadget image
```
Comment (blanquicet) [inline docs/devel/hello-world-gadget.md:210]:
```suggestion
## Signing and verifying the gadget image
```

File: docs/devel/hello-world-gadget.md [hunk 2]
```diff
@@ -207,6 +207,12 @@ Pushing ghcr.io/my-org/mygadget:latest...
...
+
+Once pushed to a container registry, you can easily sign the gadget image using [`cosign`](https://docs.sigstore.dev/signing/signing_with_containers/).
+We highly recommend you to sign your gadget image for security reasons.
+Also, if your gadget image is not signed, you will need to run it using `--verify-image=false` as we forbid running unsigned gadget images by default.
```
Comment (blanquicet) [inline docs/devel/hello-world-gadget.md:214]: What about:

```
Once you have pushed your gadget image to a container registry, it's highly recommended to sign it for security reasons. 
Tools like [cosign](https://docs.sigstore.dev/signing/signing_with_containers/) can be used for this purpose. 
Signed images ensure integrity and authenticity, adding an extra layer of trust. 
By default, Inspektor Gadget forbids running unsigned gadget images, but you can skip the verification using the `--verify-image=false` flag if necessary. However, signing your gadgets remains the preferred and more secure approach.

For more details on the verification process, refer to the [verification documentation](https://github.com/inspektor-gadget/inspektor-gadget/blob/main/docs/getting-started/verify.md#verify-image-based-gadgets). 
```

**Review by blanquicet — COMMENTED** [review]
Thanks for adding this reference!

**Review by blanquicet — APPROVED** [review]
LGTM!

File: docs/devel/hello-world-gadget.md [hunk 3]
```diff
@@ -207,6 +207,15 @@ Pushing ghcr.io/my-org/mygadget:latest...
...
 ```
 
+## Signing the gadget image
+
+Once you have pushed your gadget image to a container registry, it's highly recommended to sign it for security reasons.
+Tools like [cosign](https://docs.sigstore.dev/signing/signing_with_containers/) can be used for this purpose.
+Signed images ensure integrity and authenticity, adding an extra layer of trust.
+By default, Inspektor Gadget forbids running unsigned gadget images, but you can skip the verification using the `--verify-image=false` flag at your own risks.
+
+For more details on the verification process, refer to the [verification documentation](https://github.com/inspektor-gadget/inspektor-gadget/blob/main/docs/getting-started/verify.md#verify-image-based-gadgets).
```
Comment (mauriciovasquezbernal) [inline docs/devel/hello-world-gadget.md:217]: Do we need to use a full link here? It'll break the versioning as this will always be pointing to main.

**Pull request merged**